drivers/Kconfig

```diff
@@ -12,6 +12,7 @@ source "drivers/bluetooth/Kconfig"
 source "drivers/cache/Kconfig"
 source "drivers/can/Kconfig"
 source "drivers/clock_control/Kconfig"
+source "drivers/charger/Kconfig"
```

Comment:
Let's put this one before `clock_control` so we keep this file alphabetized.

Reply:
Ack